draft-ietf-opsec-probe-attribution.md

```diff
@@ -198,7 +209,7 @@ Using either the out-of-band or in-band technique, or even both combined, highly
 3. In-band only
 4. None
 
-Both out-of-band and in-band combined should be preferred. It could be used as an indirect means of "authenticating" the probe description URI in the in-band probe, thanks to a correlation with the out-of-band technique (e.g., a reverse DNS lookup). However, the out-of-band technique might not be possible due to several conditions: the presence of a NAT, too many endpoints to run a web server on (e.g., RIPE Atlas probes), dynamic source addresses, etc. In that case, the in-band solution should be preferred.
+Both out-of-band and in-band combined should be preferred. It could be used as an indirect means of "authenticating" the probe description URI in the in-band probe, thanks to a correlation with the out-of-band technique (e.g., a reverse DNS lookup). However, the out-of-band technique might not be possible due to several conditions: the presence of a NAT, too many endpoints to run a web server on, no access to the source (e.g., RIPE Atlas {{RIPE_ATLAS}} probes), dynamic source addresses, etc. In that case, the in-band solution should be preferred.
```

Comment:
s/no access to the source (e.g., RIPE Atlas {{RIPE_ATLAS}} probes)/, the probe source IP address cannot be known (e.g., RIPE Atlas {{RIPE_ATLAS}} probes are sent from IP addresses not owned by the probe researcher)/